script/Arcueid/Day 8/08_00_ARC08_1.txt

```diff
@@ -121,22 +124,22 @@ I felt like I had seen some sort of nostalgic dream.
 [sha:f8e2f2a3ff179d49b5aa0825373f85f0acdd5e1d]{
 -- Page 6, Offset 19687.
 -- 目を覚ました途端、<厭|いや>なモノが視界に入ってくる。
-The moment I wake up, a gruesome thing enters my field of vision.
+As soon as I wake up, something unpleasant enters my view.
 }
 [sha:b87f91d0c5637288f4f25b09e6610eb3c53b65dd]{
 -- Page 7, Offset 19688.
 -- こめかみの肉と骨をえぐるような、鋭い頭痛。
-A sharp headache that gouges onto the flesh and bones of my temple.
+A sharp headache that feels like it's gouging out the flesh and bones of my temple.
```

Comment:
maybe something like:
A sharp headache gouges deep into the flesh and bone beneath my temple,